.github/workflows/pipeline_run.yml

```
@@ -21,4 +21,7 @@ jobs:
         source env/bin/activate
         pip install -e .
         echo "Running pipeline"
-        boreholes-extract-all -l -i example/example_borehole_profile.pdf -o example/ -p example/predictions.json -m example/metadata.json -g example/example_groundtruth.json -pa all
\ No newline at end of file
+        boreholes-extract-all -l -i example/example_borehole_profile.pdf -o example/ -p example/predictions.json -m example/metadata.json -g example/example_groundtruth.json -pa all
+
+        echo "Running scoring script"
+        boreholes-score --ground-truth-path example/example_groundtruth.json --predictions-path example/predictions.json
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)